cea/datamanagement/format_helper/cea4_migrate_db.py

```
@@ -0,0 +1,451 @@
+
+    # Loop through each sheet and save as a CSV
+    for sheet_name in excel_data.sheet_names:
+        try:
+            df = pd.read_excel(path_excel, sheet_name=sheet_name)
+            # Rename columns based on the rename_dict
+            if rename_dict:
+                df.rename(columns=rename_dict, inplace=True)
+
+            # Handle the special case of renaming
+            if sheet_name == 'CONSTRUCTION':
+                output_path = os.path.join(directory_csv, f"MASS.csv")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)